app/src/main/res/drawable/button_default_highlight.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <corners android:radius="3dp" />
+    <solid android:color="#E6CCCCCC" />
```

Comment:
Should we assign these color values to resources to allow for theming in the future?

Reply:
Good idea, I will do that so we're 0.1% closer to a hotdog theme.